src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -614,6 +627,14 @@ namespace winrt::TerminalApp::implementation
             }
         });
 
+        if (debugConnection)
+        {
+            TermControl newControl{ settings, debugConnection };
+            _RegisterTerminalEvents(newControl, *newTabImpl);
+            // Split (auto) with the debug tap.
+            newTabImpl->SplitPane(SplitState::Automatic, profileGuid, newControl);
```

Comment:
This won't work during startup with a bunch of commands (I think we'll just evaluate the `Automatic` as `Vertical`), but considering this is a debug tool I'm okay with that